frontend/src/components/SocialMediaButton/index.js

```diff
@@ -1,59 +1,42 @@
+import { PropTypes } from "prop-types";
 import styles from "./index.module.css";
+import * as constants from "constants/Subvisual.js";
 
-import Github from "assets/imgs/socialButtons/github.png";
-import LinkedIn from "assets/imgs/socialButtons/linkedin.png";
-import Behance from "assets/imgs/socialButtons/behance.png";
-import Facebook from "assets/imgs/socialButtons/facebook.png";
-import Instagram from "assets/imgs/socialButtons/instagram.png";
-import Dribbble from "assets/imgs/socialButtons/dribble.png";
-import Medium from "assets/imgs/socialButtons/medium.png";
-import Twitter from "assets/imgs/socialButtons/twitter.png";
-
-const socialMediaButton = ({ name, username }) => {
+const SocialMediaButton = ({ name }) => {
```

Comment:
This should be a `SocialMediaLink` pr `SocialLink`.

Make sure to rename the files and imports too.

Reply:
I already fixed it on the other PR